src/types.ts

```diff
@@ -748,7 +748,32 @@ export interface Session {
    *  @return { Object }
    *
    */
-  recordAttribution(type: EventType, target: string, ref: string, refSrc: string): object
+  recordAttribution(type: EventType, target: string, ref: string, refSrc: string): Attribution
+}
+
+declare const refTypes: readonly ["email", "imgrec", "rec", "api", "oc", "cmp", "os"];
```

Comment:
Why not
`type EventRefType = "email" | "imgrec" | "rec" | "api" | "oc" | "cmp" | "os"`?